env_vars

```diff
@@ -0,0 +1,14 @@
+readonly HOME_DIR="$(pwd)"
```

Comment:
If you named this file with a `.env` suffix, editors such as VS Code could display them more conveniently, I think. :-)

Reply:
It's just a source file. Same approach as in smalltalkCI. Maybe I should rename it? It's not even "export". Hmm...

Reply:
I'd vote for renaming it. For instance, I do not get shellcheck or syntax highlighting in VS Code if the file does not have a suffix.

Reply:
I think just `env` is a _very_ popular choice.

Reply:
Alright, let's measure this! :-)

| glob pattern | hits on public GitHub repositories |
| --- | --: |
| `env` | [2.8k](https://sourcegraph.com/search?q=context:global+file:/env%24+count:100000&patternType=literal) |
| `*.env` | [29.1k](https://sourcegraph.com/search?q=context:global+file:/.*%5C.env%24+count:100000&patternType=literal) |
| `?*env` | [12.5k](https://sourcegraph.com/search?q=context:global+file:/.*%5B%5E.%5Denv%24+count:100000&patternType=literal) |
| `env_*` | [0.635k](https://sourcegraph.com/search?q=context:global+file:/env_%5B%5E./%5D%2B%24+count:100000&patternType=literal) |

So I guess anything but a file name ending in `_env` would be "popular". And VS Code should probably detect just `env` as environment variables, too. :D

Reply:
Just because it is hacktober, FYI: https://github.com/mikestead/vscode-dotenv/pull/28 🙃